### ISSUE TITLE
"Powiadomienia o narażeniu na kontakt COVID-19" - brak synchronizacji

### ISSUE THREAD

**Issue by tibod** [ProteGO-Safe/android#162]
**Describe the bug**
Aplikacja przestała weryfikować potencjalne ryzyko narażenia na kontakt. 
Aplikacja aktualna (aktualizacje automatyczne, ale potwierdzone też manualnie).
Na początku ryzyko było weryfikowane cyklicznie, ale po kilku miesiącach przestało. 
 
**To Reproduce**
W Ustawieniach (systemu) > Google > "Powiadomienia o narażeniu na kontakt COVID-19" data ostatniego sprawdzenia to 31 października 03:28. 

**Current behavior**
Brak synchronizacja

**Expected behavior**
Albo po ktorymś update aplikacja przestała komunikować sprawdzenie, ale z jakiegoś powodu przetała sprawdzać. 

**Screenshots**

**Smartphone (please complete the following information):**
Asus Zenfone 5, Android 10, Bluetooth włączony cały czas, dostęp do internetu przez dane mobilne i WiFi. 
Restart telefonu nic nie daje. 
Wyłączenie i ponowne włącznie "Powiadamiania" też nic nie daje.

**Additional context**

**Comment by olekstomek** [comment]
> W Ustawieniach (systemu) > Google > "Powiadomienia o narażeniu na kontakt COVID-19" data ostatniego sprawdzenia to 31 października 03:28.

Nie zauważyłem takiego problemu (inny sprzęt). W powiadomieniu o narażeniu na kontakt COVID-19 jaką masz liczbę weryfikacji ryzyka narażenia na kontakt? Zero kontroli w ostatnim czasie (14 dni)?

**Comment by tibod** [comment]
Od wspomnianego 31 października, cały czas 0.

Z poważaniem,
Aleksander Droś
790-027-848


pt., 4 gru 2020 o 23:28 olekstomek <notifications@github.com> napisał(a):

**Comment by pkleczko** [comment]
@tibod możesz zerknąć w Ustawienia i podać numer wersji EN (ten numerek z dołu ekranu jak na screenshocie poniżej "18204713000"):

![image](https://user-images.githubusercontent.com/910035/101994808-ad617000-3cc5-11eb-9597-a552f0d10dc1.png)

**Comment by tibod** [comment]
U mnie jest inna wartość: 18204516000.
Sama aplikacja jest aktualna, a przynajmniej nie widzę opcji aktualizacji.

sob., 12 gru 2020, 22:04 użytkownik Pawel Kleczkowski <
notifications@github.com> napisał:

**Comment by olekstomek** [comment]
> U mnie jest inna wartość: 18204516000.

Dokładnie mam tę samą wartość i problem nie występuje.

**Comment by pkleczko** [comment]
ok mamy pomysł na próbę poprawy tego przez wymuszenie ponownych sprawdzeń. w najbliższej wersji 4.9.0, która ukaże się przed świętami wdrożymy i poproszę Cię @tibod o sprawdzenie

**Comment by piojan** [comment]
Czy powyższa funkcjonalność ręcznego wymuszenia sprawdzenia finalnie trafiła do 4.9.0? Nie mogę jej nigdzie zlokalizować a również mam problem z brakiem weryfikacji. Ostatnie z: 26 stycznia, 16 stycznia, 12 stycznia.

**Comment by pkleczko** [comment]
@piojan tak, nie trzeba jej specjalnie aktywować, przy każdym uruchomieniu aplikacji sprawdzane jest czy zadanie w tle odpowiadające za cykliczne sprawdzanie kluczy jest uruchomione czy nie. Czy może dać znać jaki model telefonu posiadasz, z którą wersją Androida i wersją EN (jak wyżej)? Wiemy, że w niektórych modelach telefonów wystepuje taki problem (system bardzo aktywnie ubija aktywności w tle, i to pomimo wysokiego priorytetu nadawanego przez Google), takie problemy występują w aplikacjach z innych państw EU.

**Comment by olekstomek** [comment]
https://github.com/ProteGO-Safe/specs/issues/251

Ostatnie sprawdzenie z 19 maja. Smartfon w 99% czasu włączony i online.
![1622839104519](https://user-images.githubusercontent.com/26818304/120860461-02e84c80-c586-11eb-92b3-06394f741cc1.jpg)
![1622838827710](https://user-images.githubusercontent.com/26818304/120860499-0da2e180-c586-11eb-8cbd-f8458a662822.jpg)

POCO X3 NFC, Android 10, MIUI 12.0.8 (oszczędzanie baterii "zalecane" dla aplikacji - w innym Xiaomi na takim ustawieniu sprawdzanie działa prawidłowo).
Numer wersji EN v1.8 (212002104)
Problem w wersji 4.11.0 oraz po aktualizacji wczoraj do 4.12.0 bez poprawy.

**Comment by olekstomek** [comment]
POCO X3 NFC, po upgradzie do Androida 11 od 3-4 dni jest OK.
